### ISSUE TITLE
No reports if server is offline during attack

### ISSUE THREAD

**Issue by MoogyG** [threatstream/mhn#143]
First awesome job, thanks.
I have installed dionaea on raspberry.
MHN Server is running on a vagrant Ubuntu virtual machine.
I attack the sensor from a Kali raspberry using nmap, and attack is detected on the sensor page. Works well.
But if I do the same thing while server is shutted down, then start the server and wait few minutes, there is no more attack reported by sensor.
Is it normal? Server must be online during attack to be reported? No database on sensor and synchronisation with the server?

**Comment by sidra-asa** [comment]
Can you show the result of supervisorctl status ? Maybe some processes are out of work.

**Comment by MoogyG** [comment]
Ok I publish that these days

**Comment by jt6211** [comment]
Yes, that is right, if the server is not online, then the attacks will not be recorded.  MHN leverages hpfeeds events for collecting the data which is only a realtime transport mechanism.

**Comment by MoogyG** [comment]
Thanks for the reply Jason.
Is it planned to get working offline?
Can I install MHN Server on a raspberry pi? I will check that soon.

**Comment by gregcmartin** [comment]
Hi MoogyG,

You can but it's not recommended as MongoDB can take significant memory
over time.  We recommend a minimum 4g of memory for the MHN Server.

Greg

**Comment by MoogyG** [comment]
Hi Greg, you are right I've just checked hardware recommandations. I don't know MongoDB at all, Hungry stuff! 
In fact my goal is just to get a Honey pot on my home LAN to ensure there is no problem since I opened few ports on my internet router for an homemade camera system with raspberry.
I think that 4GB is the maximum that could be used when there is a big traffic with a lot of Honey pot connected no? So my MHN server load should be poor or MogoDB consumes a lot of RAM even with no work ?
Thanks again for your work guys
